{"uuid": "ec44b448-e325-4f70-9d04-da3b459c2fc3", "children": ["4ffedf81-efe0-4796-8c26-d9c174d25cff"], "befores": [{"name": "init_login", "status": "broken", "statusDetails": {"message": "TypeError: 'WebDriver' object is not callable\n", "trace": "  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\T\\Desktop\\毕设\\NWU_Campus_UI_Auto_Test\\script\\test_add_topic.py\", line 16, in init_login\n    driver().get(TOPIC_URL)\n    ^^^^^^^^\n"}, "start": 1738419952168, "stop": 1738419954730}], "start": 1738419952168, "stop": 1738419954899}